linebot/v3/audience/api/manage_audience.py

```diff
@@ -56,6 +57,8 @@ def __init__(self, api_client=None):
         if api_client is None:
             api_client = ApiClient.get_default()
         self.api_client = api_client
+        self.line_base_path = "https://api.line.me"
```

Comment:
It might be confusing, but the blob client overrides this. There is a blob URL in `_hosts`. Using Pebble could cleanly resolve this, but it's currently not possible because we are using Mustache.

The rules for resolving hosts are as follows:
- Each function with an endpoint definition has a `_host`, which is the default value.
- If a host is specified in `Configuration.py`, it can override the aforementioned default.